### PR TITLE
Added new style to responsive section design tools

### PR DESCRIPTION
**Pull request by liudmylailchenko**

### CODE REVIEW

File: src/components/SectionDesignTools.js [hunk 1]
```diff
@@ -29,14 +40,20 @@ const NewH2 = styled(H2)`
 
 const ToolsContainer = styled.div`
   display: flex;
-  margin: 54px -20px 0 -20px;
+  flex-direction: column;
+  margin: 14px -20px 0 -20px;
```
Comment (zhukmj) [inline src/components/SectionDesignTools.js]: Can be shortened to 
```
margin: 14px -20px 0
```

File: src/components/SectionDesignTools.js [hunk 2]
```diff
@@ -29,14 +40,20 @@ const NewH2 = styled(H2)`
...
   padding-bottom: 34px;
+
+  @media screen and (min-width: 660px) {
+    flex-direction: row;
+    margin: 54px -20px 0 -20px;
```
Comment (zhukmj) [inline src/components/SectionDesignTools.js]: Use shorten form

File: src/components/SectionDesignTools.js [hunk 3]
```diff
@@ -8,19 +8,30 @@ import clock from '../images/clock.svg';
...
-  margin: 0 auto;
-  padding-bottom: 90px;
+  padding-bottom: 50px;
+
+  @media screen and (min-width: 660px) {
+    padding-bottom: 90px;
+  }
 `;
 
 const NewH2 = styled(H2)`
```
Comment (zhukmj) [inline src/components/SectionDesignTools.js]: The naming of this styled component isn't good.  Use something more meaningful for this specific section e.g. `Heading` or `SectionHeading`

**Review by zhukmj — CHANGES_REQUESTED** [review]

File: src/components/SectionDesignTools.js (hunk 1 above)
Comment (liudmylailchenko) [reply src/components/SectionDesignTools.js]: Done

File: src/components/SectionDesignTools.js (hunk 2 above)
Comment (liudmylailchenko) [reply src/components/SectionDesignTools.js]: Done

File: src/components/SectionDesignTools.js (hunk 3 above)
Comment (liudmylailchenko) [reply src/components/SectionDesignTools.js]: Done

**Review by zhukmj — APPROVED** [review]
Looks great now! Thanks @liudmylailchenko, you're awesome 😛

**Pull request merged**